src/components/LoginFields.js

```diff
@@ -14,8 +17,11 @@ const LoginFields = () => {
       name,
     }).unwrap()
       .then((response) => {
+        console.log(response);
         if (response.message === 'Logged In Successfully') {
-          navigate('/home', { replace: true });
+          dispatch(updateUser({ loggedIn: true, data: response.user }));
+          window.location.reload();
+          navigate('/', { replace: true });
         } else {
           // Handle unsuccessful login
           console.error('Invalid username');
```

Comment:
Instead of using a `console.error` to display error messages, I suggest implementing a different UI approach to let the user know what went wrong since average users will never access the console to check what the error is. As an alternative you can use an `alert`.